src/app.service.ts

```diff
@@ -2,27 +2,28 @@ import { Injectable } from '@nestjs/common';
 import fetch from 'node-fetch';
 import dotenv from 'dotenv'
 dotenv.config()
+import response from './response.json'
 
 
 @Injectable()
 export class AppService {
   async fetchStores(reqParam) {
     if (!reqParam) {
-      return {}
+      return response
     }
     try {
-      const response = await fetch(
```

Comment:
Why have we commented out this? will we be sending hard-coded response for the experience sake?

Reply:
yes